tests/common.go

```diff
@@ -410,6 +418,8 @@ func ParseFlags() {
 				StorageDriverUpgradeVersion: volUpgradeVersion,
 				StorageDriverBaseVersion:    volBaseVersion,
 				AppList:                     appList,
+				ConfigMap:                   configMapName,
```

Comment:
looks like your config map changes for security are showing up here? Let's keep this PR just for CSI changes.

Reply:
it's not needed in this PR, will delete.